src/components/ChatMessageItem.js

```diff
@@ -163,6 +175,16 @@ const ChatMessageItemComponent = ({ type, message, eva: { style, theme }, create
       </View>
     </TouchableOpacity>
   );
+
+  return (
+    <Tooltip
+      anchor={renderChatMessageIconComponent}
+      visible={tooltipVisible}
+      onBackdropPress={() => setTooltipVisible(false)}
+      placement="top">
+      <Text style={style.tooltipText}>Sent by: {message.sender.name}</Text>
```

Comment:
@iamprayush The app will crash if the message sender is empty (Some messages doesn't have sender details). So show tooltip only for messages which contain sender details.

Reply:
I've added the tooltip conditionally now, PTAL!